docs/source/application_development/getting_started.rst

```diff
@@ -302,7 +302,7 @@ Encrypt
    from nucypher.characters.lawful import Enrico
 
    enrico = Enrico(policy_encrypting_key=policy_encrypting_key)
-   ciphertext, signature = enrico.encrypt_message(plaintext=b'Peace at dawn.')
+   ciphertext = enrico.encrypt_message(plaintext=b'Peace at dawn.')
```

Comment:
The idea here is that the user can get the signature from the `MessageKit` if they want to, right?  Why are we calling it `ciphertext` in the docs?

Reply:
Yes, signature is either in the plaintext (if we signed the plaintext) or kept inside the message kit (if we signed the ciphertext). Now that I think about it, why can't we keep the signature outside in both cases?

> Why are we calling it ciphertext in the docs?

Honestly, I don't know. Probably some historical reasons.

Reply:
Anyway, changed to `message_kit`.